frontend/src/app/modules/common/header/logo.sass

```diff
@@ -13,5 +13,6 @@
     background-size: contain
     background-repeat: no-repeat
 
-  @include breakpoint(680px down)
+  @include breakpoint(850px down)
     display: none
```

Comment:
We could think about showing just a logo icon here instead of logo+full name, so that at least some tablet users can get back to the home page. Though in general I think we need to rethink navigation anyway, so perhaps our time is spend better somewhere else.